data/maxton4life.json

```diff
@@ -1,41 +1,41 @@
 {
   "name": "Hameed Kareem",
-  "bio": "A Self-Thought DevOps Engineer Who is Passionate in Delivering Software Products in high velocity by Designing Solutions that helps businesses grows.",
+  "bio": "A Self-Taught DevOps Engineer Who is Passionate about Delivering Software Products at high velocity by Designing Solutions that help businesses grow.",
   "tags": ["Linux", "Python", "CI/CD", "Docker", "Kubernetes"],
 
   "socials": [
     {
       "icon": "FaTwitter",
-      "url": "https://twitter.com/hameed__kareem"
+      "URL": "https://twitter.com/hameed__kareem"
     },
     {
       "icon": "FaGithub",
-      "url": "https://github.com/maxton4life"
+      "URL": "https://github.com/maxton4life"
```

Comment:
```suggestion
      "url": "https://github.com/maxton4life"
```

Reply:
Grammarly do this one 😂

I'll do this one now.